brokenspoke_analyzer/core/analysis.py

```diff
@@ -195,7 +195,7 @@ def create_synthetic_population(
     grid = gpd.GeoDataFrame(
         {
             "geometry": cells,
-            "POP10": population,
+            "POP20": population,
             "BLOCKID10": [
```

Comment:
I am assuming this should become `BLOCKID20`.

Reply:
Thanks, fixed!